docs/DeveloperGuide.adoc

```diff
@@ -239,6 +239,128 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+//tag::moduledatabaseadministration[]
+=== Module Database Administration
+
+Module Database Administration are a series of commands that administrators can use to make necessary changes to keep
+the module database up to date, or to cater to their own needs.
+
+[NOTE]
+The commands pertaining to module database admnistration can only be executed by users of type `Admin`.
+
+==== Current Implementation
+
+The Module database administration mechanism is facilitated by `ModuleList` in `Model` via the following operations:
+
+* `ModuleList#addModule(...)` -- This method adds a module to the module list.
+* `ModuleList#removeModule(...)` -- This method removes a module from the module list.
+* `ModuleList#hasModule(...)` -- This method check if the given module already exist.
```

Comment:
This method `check` if the given module already `exist`.
`check` -> `checks`
`exist` -> `exists` (referenced from https://support.google.com/a/answer/1071113?hl=en)